semconvgen/generator_test.go

```diff
@@ -0,0 +1,57 @@
+package main
+
+import (
+	"os"
+	"reflect"
+	"testing"
+)
+
+func TestCapitalizations(t *testing.T) {
+	tests := []struct {
+		name            string
+		capitalizations string
+		expected        []string
+	}{
+		{
+			name:            "No additional capitalizations",
+			capitalizations: "",
+			expected:        staticCapitalizations,
+		},
+		{
+			name:            "Some additional capitalizations",
+			capitalizations: "ASPNETCore\nJVM",
```

Comment:
Can we add a test case when there would be many blank lines?
People usually add EOL at the end of text files.